README.md

```diff
@@ -67,10 +67,21 @@ Launch the application with the following commands:
     ros2 launch hello_world_simulation empty_world.launch.py
     ```
 
+- *Running Simulation Application With Visualization*
+    ```bash
+    export DISPLAY=:0
```

Comment:
I would assume that the user has configured the display appropriately and don't see the need to set this explicitly here